README.md

```diff
@@ -18,110 +21,61 @@ Learn more about [mixed reality](https://developer.microsoft.com/en-us/windows/m
-
-<img src="External/ReadMeImages/MotionControllerTest_Teleport.png" width="500px">
-
-If you're looking for **Controller models**:
-* See the [**MotionControllerTest**](Assets/HoloToolkit-Examples/Input/Scenes/MotionControllerTest.unity) scene.
-* See:
-    * ControllerVisualizer.cs
-    * ControllerInfo.cs
-    * GLTFComponentStreamingAssets.cs
-    * The entire Utilities\Scripts\GLTF folder.
-* **IMPORTANT** Requires the **10.0.16299.0 SDK**, or you will not be able to build these scripts.
-    - You can install the SDK using the Visual Studio Installer.
-* **IMPORTANT** Currently, motion controller's GLTF 3D model is only visible when you deploy through Visual Studio. In Unity's game mode, you should assign override model. <img src="External/ReadMeImages/MotionControllerTest_ModelOverride.png" width="700px">
-
-If you're looking for **teleporting**:
-* See the [**MotionControllerTest**](Assets/HoloToolkit-Examples/Input/Scenes/MotionControllerTest.unity) scene.
-* Controls are the same as the Cliff House, using either an Xbox controller or motion controllers. Thumbstick up for teleport, down for backup, left/right for rotation.
-* See:
-    * MixedRealityTeleport.cs
-    * MixedRealityCameraParent.prefab
-    * TeleportMarker.prefab
-    
-If you're looking for **Xbox Controller Input** via the MixedRealityToolkit's InputManager:
-* See the [**XboxControllerExample**](Assets/HoloToolkit-Examples/Input/Scenes/XboxControllerExample.unity) scene.
-    
-If you're looking for **Boundary** tools:
-* See the [**BoundaryTest**](Assets/HoloToolkit-Examples/Boundary/Scenes/BoundaryTest.unity) scene.
-* The **Boundary** folder has the scripts that support defining the floor for your immersive applications.
-* The scripts help to draw the floor for immersive headsets and also allows you to check if an object is within those bounds.
-* See:
-    * BoundaryManager.cs
-    * MixedRealityCameraParent.prefab
-    
-If you're looking for **Motion controller Grab Mechanics**:
-* See  [**MotionControllersGrabMechanics**](https://github.com/Microsoft/MixedRealityToolkit-Unity/tree/master/Assets/HoloToolkit-Examples/MotionControllers-GrabMechanics) page.
-<img src="External/ReadMeImages/MRTK_MotionController_GrabMechanics.jpg" width="500px">
-    
+# Examples and QuickStart scenes
+The MRTK includes many great samples and starter scenes to demonstrate the uses of the MRTK API, these include:
+
+| [![Keyboard input sample](External/ReadMeImages/InputKeyboard.png)](/Assets/HoloToolkit-Examples) [Keyboard input sample](/Assets/HoloToolkit-Examples) | [![Input manager tests](External/ReadMeImages/InputManagerTest.png)](/Assets/HoloToolkit-Examples) [Input manager tests](/Assets/HoloToolkit-Examples) | [![Interactive button demos](External/ReadMeImages/InteractiveButtonDemo.png)](/Assets/HoloToolkit-Examples) [Interactive button demos](/Assets/HoloToolkit-Examples) |
```

Comment:
This is an awesome addition, but I kinda wish all the images were the same size.

It's not big deal.

Reply:
The images were as close as I could make them to the same size.  Will have another look

Reply:
It's not big deal.  we can skip this.